docs/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+EDF = "ccffbfc1-f56e-50fb-a33b-53d1781b2825"
```

Comment:
```suggestion
```
The documentation step in CI ensures that the right version of EDF.jl is added to the environment. If you have it listed as a project dependency and you inadvertently `instantiate` before you `develop`, you can end up in a weird state where the released version of EDF.jl is used instead of the checked out one, IIRC. Maybe that's not entirely accurate, I don't fully recall, but the general practice is to not include it here.

Reply:
Sweet, that makes sense!